apps/scan/frontend/src/screens/scanner_double_feed_calibration_screen.tsx

```diff
@@ -98,7 +98,11 @@ export function ScannerDoubleFeedCalibrationScreen(): JSX.Element | null {
         return (
           <CalibrationScreen>
             <FullScreenMessage
-              title="Calibration Failed"
+              title={
+                status.error === 'double_feed_calibration_timed_out'
+                  ? 'Calibration Timed Out'
+                  : 'Calibration Failed'
+              }
```

Comment:
@jonahkagan unfortunately I don't have a PDI scanner at home so I don't have screenshots for this yet. You can imagine the change, generate them yourself, or wait until later in the week when I have them.

Reply:
No worries, I can imagine :)